mp/src/game/client/momentum/ui/SettingsPanel/MomentumSettingsDialog.cpp

```diff
@@ -71,7 +71,7 @@ class GroupPanel : public Panel
     int m_iSpacingX;
 };
 
-
+static ConVar mom_settings_remember_tab("mom_settings_remember_tab", "1", FCVAR_CLIENTCMD_CAN_EXECUTE | FCVAR_SERVER_CAN_EXECUTE, "Remembers last opened tab in settings");
```

Comment:
Use MAKE_TOGGLE_CONVAR from `mom_shareddefs.h`, and make sure to archive the setting so it applies between loads:

```suggestion
static MAKE_TOGGLE_CONVAR(mom_settings_remember_tab, "1", FCVAR_ARCHIVE, "Toggles remembering the last opened tab in settings. 0 = OFF, 1 = ON.\n");
```